desktop/src/main/java/bisq/desktop/main/dao/wallet/send/BsqSendView.java

```diff
@@ -296,18 +296,25 @@ private void addSendBtcGroup() {
                     Transaction txWithBtcFee = btcWalletService.completePreparedSendBsqTx(preparedSendTx, true);
                     Transaction signedTx = bsqWalletService.signTx(txWithBtcFee);
                     Coin miningFee = signedTx.getFee();
-                    int txSize = signedTx.bitcoinSerialize().length;
-                    showPublishTxPopup(receiverAmount,
-                            txWithBtcFee,
-                            TxType.INVALID,
-                            miningFee,
-                            txSize, receiversBtcAddressInputTextField.getText(),
-                            btcFormatter,
-                            btcFormatter,
-                            () -> {
-                                receiversBtcAddressInputTextField.setText("");
-                                btcAmountInputTextField.setText("");
-                            });
+
+                    if (miningFee.getValue() > receiverAmount.getValue())
```

Comment:
I think it should be >=

Reply:
I think a check that the output is no below dust limit is in the wallet service, but maybe better to cross check?

Reply:
Yes, we probably should already prevent it, if the amount will be ```0``` in the end. Maybe we should also prevent it already before, but there it is hard where to draw a line.

Reply:
> I think a check that the output is no below dust limit is in the wallet service, but maybe better to cross check?

There is a dust check in place already.